source/text.rst

```diff
@@ -34,16 +34,17 @@ Questions
 ---------
 
 
-1. Find practical uses cases of where you'd like to manipulate text.
-2. Using the above documentation and the interpreter interactively try to map
-   those use cases to actual code solutions.
-3. Try to imagine use cases for each of the methods that exist on 
-   `str` objects in the docs.
+1. Using the documentation, how would you make a string all capitalized?
+2. Given the string `"I like chocolate"`, how would you change the word chocolate to something
+   else?
+3. What other ways would you want to manipulate a string?
+4. Try experimenting with the other methods that exist in the docs. How would you use the each
+   of those methods to manipulate a string? Use the interpreter interactively to try them out.
 
```

Comment:
> I wonder if we should provide the most useful/important bits of the documentation, like basic string operations as print-outs. I know that's really old-school (no pun intended).

I really want to encourage the students to find the most useful parts themselves. If they ask "how can I do ___ with a string" the answer should be "check the documentation".